### DISCUSSION TITLE
Concept for a new Oqtane.org homepage

### DISCUSSION THREAD

**Discussion by TonyValenti** [oqtane/oqtane.framework#804] [General]
I don't have the scaling and such down, but take a look at it with a browser width of ~1024:
https://tvfasterlaw.editorx.io/mysite-8

I'd love feedback.

**Comment by mikecasas** [comment]
Looks good!

**Comment by sbwalker** [comment]
I do like this design. The one question which keeps coming up is "when will oqtane.org be running on Oqtane?". And so far my answer has been that Oqtane is a modular application framework - not a CMS. However upon further reflection, I have come to a different conclusion. When I reflect on the website we built to manage the open source community for DNN it had some basic content - but the majority of the functionality was comprised of custom built web applications to provide advanced functionality to the community. And I can definitely envision needing similar capabilities in the Oqtane community. So in this sense, it makes perfect sense to use Oqtane to run the oqtane.org community website. It also satisfies the open source mantra of "eat your own dog food" as there is no better way to show confidence in the product than to run it in production yourself. And another added benefit which we realized with DNN was that as the community grew it exposed performance/scalability issues which we may not have been aware of otherwise. So once V2 is released I would really like to migrate the site to Oqtane.

**Reply by TonyValenti** [reply]
Do you have a designer you are working with?

I think that the DotNetNuke.com website never really looked sexy and that was a barrier in the long run. 

It would be great to be able to provide a mock-up to a designer who can really make the site look snazzy.

**Reply by sbwalker** [reply]
Obviously, beauty is in the eye of the beholder - so what one person thinks is sexy, another will think is awful ( case in point - some community members have commented that they are really impressed with the existing site ). The DNN website had many different designs in the past 20 years - the ironic thing is that adoption of DNN was highest in the years when the team could not afford professional design resources ( ie. 2003-2009 ) and declined in the years when we had funding and spent a lot of money on professional designers, marketing, brand identity, etc... That being said, I do agree that marketing/design is important - and no, I am not working with a designer currently.

**Reply by TonyValenti** [reply]
Do you have an Oqtane company/foundation set up that is able to accept revenue and invest in business operations?  If so, can I invest < $1000 to have a designer apply that look to an Oqtane homepage once V2 is available?

**Reply by sbwalker** [reply]
The .NET Foundation is the legal entity behind Oqtane. And although I am the Chair of the Project Committee I am not sure how third parties can invest in .NET Foundation projects at this point, other than via Github Sponsorship. I will do some research in this area. The simpler path for this would be for you to work with a designer ( in conjunction with the Oqtane community to get feedback ) and then submit a website design as an open source contribution.

**Reply by TonyValenti** [reply]
@sbwalker Thanks! I think that with any project, commercial or open source, properly aligning incentives and expectations sets the groundwork for the long-term success of the project and I hope that we can all learn from our past experiences with DNN.

I will wait to hear back about financials.

When I was involved with DNN, my perception was that nearly every developer using DNN was making significantly more money than you, joe, and Charles, before funding and funding only brought you guys on-par with the rest of the community (obviously I don't actually know since I never saw your paycheck).  I believe that is because incentives weren't properly aligned early on and expectations weren't fully communicated.  My perception was that this left the most important folks on the project essentially begging others in the community to pay them.  DNN had to approach its community from a position of weakness.

That should not be the case with Oqtane.

I believe that Proper monetization is an essential part of any OSS project and vital to its long term success.  And, just because something is available free, doesn't mean that incentives can't be put in place that encourage companies to make financial commitments to projects - in many industries, the companies willing to spend the most win - and you can't spend the most if you don't have it coming in.

I saw this play out first hand when I started PowerDNN.  There were a number of hosting Platforms:  Plesk, WHMCS, DotNetPanel, Helm, and a bunch of others I can't remember right now.  Plesk was one of the more expensive ones on the market and I chose them because the cost indicated two things to me:
1.  They were better.
2.  They'd be around longer.

As time went on, that decision was reaffirmed time and time again.  So many of my competitors had problems because they picked a vendor that eventually went out of business.  There were times when I wrote $250,000 checks to Parallels for Plesk licenses, and, while I wish I didn't have to spend so much, it was worth it, and I was much better off paying that kind of cash to Parallels than a fraction of that to DotNetPanel (who went defunct).

Anyways, I'd love to talk about long-term monetization off-line and setting the project up for financial success.
